packages/functions_client/lib/functions_client.dart

```diff
@@ -1,4 +1,6 @@
 library functions_client;
 
+export 'package:http/http.dart' show ByteStream;
```

Comment:
Added this, because I thought people might want to type cast the `response.data` as `ByteStream`.